### ISSUE TITLE
Cosmetic filter for translate.google.com breaks the site

### ISSUE THREAD

**Issue by bogachenko** [ryanbr/fanboy-adblock#631]
This rule `translate.google.com##.gb_g` breaks <https://translate.google.com> for me.

When i
 click on the "Google Apps" button (see 1 screenshot),
![1](https://user-images.githubusercontent.com/8535285/48230875-5c734500-e3a4-11e8-9445-8fa08b51bc28.png)

 it disappears (see 2 screenshot)
![2](https://user-images.githubusercontent.com/8535285/48230889-685f0700-e3a4-11e8-8e8f-b1193d34b2bc.png)

Check it out?

**Comment by hawkeye116477** [comment]
Instead of that,  `translate.google.com#?#.gb_g:-abp-contains(cookie)`

**Comment by smed79** [comment]
Maybe
```
translate.google.com#?#.gb_g:-abp-contains(Yes, get Chrome now)
```

**Comment by bogachenko** [comment]
@smed79 no. I am Russian, I have a google translator in Russian. Special for you (and for your rule) changed the language in google to English. result 0. all broken fuck.

@hawkeye116477 no. anyway no. nothing changed. these fagots from google created their fagot code.

**Comment by bogachenko** [comment]
so you need to delete this rule `translate.google.com##.gb_g`

**Comment by hawkeye116477** [comment]
Or this `translate.google.com###gb > div[style*="background-color"]`.

**Comment by bogachenko** [comment]
where are the moderators when they are needed?

**Comment by smed79** [comment]
> where are the moderators when they are needed?

see https://forums.lanik.us/viewtopic.php?p=66878#p66878

You can find here an example how to create your own filter list https://github.com/ryanbr/fanboy-adblock/issues/289#issuecomment-372155038

**Comment by bogachenko** [comment]
I know how to make filters .. thanks.
the question was not about that .. re-read the topic.

**Comment by krystian3w** [comment]
`translate.google.com##.gb_Xc.gb_g`?

> ![obraz](https://user-images.githubusercontent.com/35370833/48366671-dcecbb00-e6ae-11e8-9960-900c0dd38c0b.png)

**Comment by bogachenko** [comment]
It looks like this filter will not be deleted.
so.........

**Comment by krystian3w** [comment]
Delete blue bar.

**Comment by krystian3w** [comment]
You have to give it 2 weeks, more or less it goes to the repair and adding filters.

**Comment by bogachenko** [comment]
I do not have it by default. I’ve removed it by the above data filter -  `translate.google.ru##body > div[id="gb"][style="min-width: 750px;"] > div.gb_g`

taken from Adguard’s Annoyance List

**Comment by krystian3w** [comment]
`.gb_Cd` - blue color bar in CSS

 `.gb_Xc` `display: block` for `.gb_g`

**Comment by bogachenko** [comment]
No blue bar. Nothing to remove I have only one problem with a broken site

**Comment by krystian3w** [comment]
This is fix button, rule should no more cut pop-up and button...

**Comment by bogachenko** [comment]
`translate.google.com ##. gb_Cd.gb_g` - still breaks the site. nothing changed
`translate.google.com##.gb_Xc.gb_g` - still breaks the site. nothing changed

**Comment by krystian3w** [comment]
Execepted bad filter?

`translate.google.com#@#.gb_g`

If still not fix, I add to PR filter form adguard.

**Comment by krystian3w** [comment]
Google change look and filters now no cut annoyance bar "use chrome"

`https://translate.google.com/?hl=ru`

![obraz](https://user-images.githubusercontent.com/35370833/49258389-94b2f400-f435-11e8-8b6a-f3812c3cd33c.png) |
--- |

**Comment by adranger** [comment]
I don't see the bar in the new word/phrase translator design, but it still shows up when you translate a whole page: https://translate.google.com/translate?js=n&sl=auto&tl=hu&u=example.org

**Comment by krystian3w** [comment]
But I no see this bar on 

`https://translate.google.com/translate?js=n&sl=auto&tl=hu&u=example.org`

And older links, maybe removed promotion google chrome.

**Comment by krystian3w** [comment]
@adranger  you are using Opera?

* `https://translate.google.com/translate?hl=hu&sl=en&tl=hu&u=https%3A%2F%2Fexample.org%2F&hl=ru`

   ![image](https://user-images.githubusercontent.com/35370833/49369653-37979680-f6f2-11e8-874b-629cd7b0df5b.png) |
   --- |

    _(ad use Chrome)_


---

* `https://translate.google.pl/?hl=en` <br> `https://translate.google.pl/?hl=en`

   ![image](https://user-images.githubusercontent.com/35370833/49369738-6d3c7f80-f6f2-11e8-8b65-26f97b00a62f.png) |
    --- |

   _(privacy policy (cookies / GDPR))_

**Comment by adranger** [comment]
Firefox. Anyway, I think that they're going to change the design for full webpage translations too, so there's no need to spend any more time on it.

**Comment by krystian3w** [comment]
The worst thing is that these stripes do not differ by anything other than the background color in the style attribute.
Blue has the color given and black ends without color value '`:"`' for property "`background-color`".
